common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -734,6 +792,12 @@ protected TokenResult performSilentTokenRequest(
 
         final TokenRequest refreshTokenRequest = strategy.createRefreshTokenRequest(parameters.getAuthenticationScheme());
         refreshTokenRequest.setClientId(parameters.getClientId());
+        if (parameters.getChildClientId() != null) {
```

Comment:
Add null or empty check and consistent with authorization builder, where you check for both child redirect uri and child client id?

Reply:
Added a method in ComandParameters to check for these parameters so that it is consistent across all classes